stubs/3.2/queue.pyi

```diff
@@ -6,5 +6,6 @@ from typing import Any
 
 class Queue():
     def get(self, block: bool = True, timeout: float = None) -> Any: pass
+    def put_nowait(self, item: Any) -> None: pass
```

Comment:
While this works, I think you could give the class a type parameter and make that the type of `item` and the return type of `get`.